tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx

```diff
@@ -28,61 +55,85 @@ describe('SubjectsStep test with some data', () => {
     expect(addSubject).toBeInTheDocument()
 
-    fireEvent.click(firstField)
-    fireEvent.change(firstField, { target: { value: 'Languages' } })
-    fireEvent.keyDown(firstField, { key: 'ArrowDown' })
-    fireEvent.keyDown(firstField, { key: 'Enter' })
+    const firstField = screen.getAllByLabelText(
+      'becomeTutor.categories.mainSubjectsLabel'
+    )
+
+    fireEvent.click(firstField[0])
+    fireEvent.change(firstField[0], {
+      target: { value: 'Category 2' }
+    })
+    fireEvent.keyDown(firstField[0], { key: 'ArrowDown' })
+    fireEvent.keyDown(firstField[0], { key: 'Enter' })
+
+    expect(firstField[0].value).toBe('Category 2')
 
     const secondField = screen.getByLabelText(
-      /becomeTutor.categories.subjectLabel/i
+      'becomeTutor.categories.subjectLabel'
     )
 
-    fireEvent.click(secondField)
-    fireEvent.change(secondField, { target: { value: 'Danish' } })
-    fireEvent.keyDown(secondField, { key: 'ArrowDown' })
-    fireEvent.keyDown(secondField, { key: 'Enter' })
+    fireEvent.click(await secondField)
+    fireEvent.change(await secondField, {
+      target: { value: 'Subject 2' }
+    })
+    fireEvent.keyDown(await secondField, { key: 'ArrowDown' })
+    fireEvent.keyDown(await secondField, { key: 'Enter' })
+
+    expect(secondField.value).toBe('Subject 2')
+
     fireEvent.click(addSubject)
+
+    const chip = await screen.findByTestId('chip')
+    expect(chip).toBeInTheDocument()
   })
 
-  it('should add a new subject', async () => {
+  it('should delete a subject', async () => {
     const addSubject = screen.getByTestId('add-subject')
-    const firstField = screen.getByLabelText(
-      /becomeTutor.categories.mainSubjectsLabel/i
-    )
 
     expect(addSubject).toBeInTheDocument()
 
-    fireEvent.click(firstField)
-    fireEvent.change(firstField, { target: { value: 'Languages' } })
-    fireEvent.keyDown(firstField, { key: 'ArrowDown' })
-    fireEvent.keyDown(firstField, { key: 'Enter' })
+    const firstField = screen.getAllByLabelText(
+      'becomeTutor.categories.mainSubjectsLabel'
+    )
+
+    fireEvent.click(firstField[0])
+    fireEvent.change(firstField[0], {
+      target: { value: 'Category 2' }
+    })
+    fireEvent.keyDown(firstField[0], { key: 'ArrowDown' })
+    fireEvent.keyDown(firstField[0], { key: 'Enter' })
+
+    expect(firstField[0].value).toBe('Category 2')
 
     const secondField = screen.getByLabelText(
-      /becomeTutor.categories.subjectLabel/i
+      'becomeTutor.categories.subjectLabel'
     )
 
-    fireEvent.click(secondField)
-    fireEvent.change(secondField, { target: { value: 'English' } })
-    fireEvent.keyDown(secondField, { key: 'ArrowDown' })
-    fireEvent.keyDown(secondField, { key: 'Enter' })
+    fireEvent.click(await secondField)
+    fireEvent.change(await secondField, {
+      target: { value: 'Subject 2' }
+    })
+    fireEvent.keyDown(await secondField, { key: 'ArrowDown' })
+    fireEvent.keyDown(await secondField, { key: 'Enter' })
```

Comment:
I guess that await doesn't need here.

Reply:
No, it is needed. The test will not work without it.